src/api/submission/validations.py

```diff
@@ -875,7 +866,7 @@ def validate_fish_family_subset(self):
         if isinstance(fish_family_subset, list) is False:
             return self.ok(self.identifier)
 
-        fish_attribute_ids = set([ob.get("fish_attribute") for ob in obs])
+        fish_attribute_ids = {ob.get("fish_attribute") for ob in obs}
```

Comment:
Again I'm unfamiliar... this is a dict comprehension with keys only? Where do the vals come from?